api/src/controllers/changes.js

```diff
@@ -312,11 +252,9 @@ const filterPurgedIds = feed => {
 };
 
 const processRequest = (req, res) => {
-  return initFeed(req, res)
-    .then(feed => {
-      // don't return promise - could be a longpoll request
-      getChanges(feed);
-    });
+  return initFeed(req, res).then(feed => {
+    return getChanges(feed);
+  });
```

Comment:
Nit: Good chance to use the auto return arrow function.

```suggestion
  return initFeed(req, res)
    .then(feed => getChanges(feed));
```